### PR TITLE
Updated screens and MD file with new Show/Hide recommendation for Adv…

### PR DESCRIPTION
**Pull request by wrnkld**
…anced Options.

## Description
Write a few sentences describing the overall goals of the pull request's commits.

(optional) Include related PRs, Issues and or Jira stories.

## Changes

Write a list of changes the PR introduces

* Change 01
* Change 02
* Change 03

## New Pattern Checklist (if relevant)

- [ ] Pattern does not already exist
- [ ] Pattern solves common problems
- [ ] Pattern has community interest (You've addressed the PatternFly mailing list)
- [ ] New files are in a folder titled with the name of the pattern
- [ ] All images are in an "img" folder and embedded images have the correct filepath
- [ ] Both overview.md and design.md files are present

### CODE REVIEW

**Comment by serenamarie125** [comment]
LGTM!  Thanks @wrnkld

File: pattern-library/forms-and-controls/expand-collapse-section/design/design.md [hunk 1]
```diff
@@ -4,16 +4,17 @@
...
 
 2. **Divider Line:** The divider line is used to divide the two distinct sections.
 <br><br>
-**Please note: if there is not a specific grouping label, "Advanced Options" is a possible generic label option. If there are more specific labels that help users to understand the context of what is being hidden then it is recommended that you use those labels to improve usability.*
+**Please note: if there is not a specific grouping label, "Show Advanced Options" is a possible generic label option. If there are more specific labels that help users to understand the context of what is being hidden then it is recommended that you use those labels to improve usability.*
 
 ## Expanded Section
 ![Image of expanded section](img/advancedoptions_example2.png)
 
 1. **Caret Icon:** When expanded, the caret icon will be facing down.
```
Comment (LHinson) [inline pattern-library/forms-and-controls/expand-collapse-section/design/design.md]: Consolidate the label names to just say "Caret Icon and Label" vs being two separate things.

**Comment by LHinson** [comment]
Given our latest approach re: accessibility (defining overarching guidelines vs per component notes, I think you can just remove the accessibility section)

**Comment by catrobson** [comment]
Design updates look good to me.

**Comment by LHinson** [comment]
My comments have been addressed. Thanks for this contribution @wrnkld! I'll merge it.

**Pull request merged**